container-conf/build.sh

```diff
@@ -8,3 +8,15 @@ build_as_root() {
     build_yum config-manager --set-enabled google-chrome
     build_yum install google-chrome-stable
 }
+
+build_as_run_user() {
+    umask 022
+    cd ~
+    git clone https://github.com/radiasoft/sirepo.git
+    cd sirepo
+    declare -a d=($(jq -r '.devDependencies | keys |  @sh' package.json))
+    for n in "${d[@]}"
+    do
+      npm install -g $(eval n=$n && echo $n)
```

Comment:
eval is generally a sign that you need to tread carefully and try to find a different way of doing things. For example, I don't know what the restrictions are on npm package names but maybe `; env` is a valid package name. If you do
```bash
n='\; env'
npm install -g $(eval n=$n && echo $n)
>>> 8.12.1
```
I was hoping that might print out our entire env which would be a security hole. But, instead it prints out 8.12.1 which I believe is the version of npm I have installed. Not terrible but not what we want the code to do. If I was more clever I imagine I could craft a package name that would do something worse.

So, we need to find a way to do this w/o eval. jq is a pain so that isn't easy. I tried a few different things but I think this might be the best option:

```bash
    local -a d=($(jq -r '.devDependencies | keys[] |  @sh' package.json))
    npm install -g "${d[@]}"
```

@robnagler can you add your thoughts? I'm worried there may be some word splitting / expansion problems but I'm hopeful with the quotes it gets around them...

One final issue is that the correct versions aren't installed. The package.json specifies version of the packages (currently all "latest" but it hasn't always been that way). So, we need to install the correct version of each package too (ex `npm install -g karma@latest`). I imagine jq has a way to get the values as well as the keys. So, you could get the two lists and either zip them together somehow (not sure if bash has a feature for this) or iterate with and index and index into both lists. Happy to brainstorm how to do it.